server/models/Profile.js

```diff
@@ -0,0 +1,52 @@
+const Schema = mongoose.Schema;
```

Comment:
This model included but it's should be fine since i will use Rebase Merge when #3 already merged